comp/core/tagger/taggerimpl/collectors/workloadmeta_main.go

```diff
@@ -159,10 +160,11 @@ func (c *WorkloadMetaCollector) stream(ctx context.Context) {
 // NewWorkloadMetaCollector returns a new WorkloadMetaCollector.
 func NewWorkloadMetaCollector(_ context.Context, store workloadmeta.Component, p processor) *WorkloadMetaCollector {
 	c := &WorkloadMetaCollector{
-		tagProcessor:           p,
-		store:                  store,
-		children:               make(map[string]map[string]struct{}),
-		collectEC2ResourceTags: config.Datadog().GetBool("ecs_collect_resource_tags_ec2"),
+		tagProcessor:                      p,
+		store:                             store,
+		children:                          make(map[string]map[string]struct{}),
+		collectEC2ResourceTags:            config.Datadog().GetBool("ecs_collect_resource_tags_ec2"),
+		collectPersistentVolumeClaimsTags: config.Datadog().GetBool("kubernetes_persistent_volume_claims_as_tags"),
```

Comment:
Could we refactor the code to rather than using the `pkg/config` here we use the config component? I'm sure we can find a way to pass it down from the `comp/core/tagger/taggerimpl/tagger.go` to here.

Reply:
This is something that we definitely need to do, but given that there are multiple `config.Datadog()` usages inside the tagger directories that should use the Config component instead, I suggest to leave it for a future PR (done by container-platform team). Otherwise, we will introduce lots of unrelated changes on this one.

Reply:
Ok for me, creating card 👍